Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/02-state-storage.md

```diff
@@ -1,38 +1,37 @@
 # Use the Azure CLI to create a storage account to store the Terraform state files.
 This storage account will be used to store the state of each deployment step and will be accessed by Terraform to reference values stored in the various deployment state files.
 
-Create some variables to start with
-
-```bash
-REGION=<REGION>
-STORAGEACCOUNTNAME=<UNIQUENAME>
-CONTAINERNAME=akscs
-TFSTATE_RG=tfstate
-```
-
-or with Powershell
-```powershell
-$REGION=<REGION>
-$STORAGEACCOUNTNAME=<UNIQUENAME>
-$CONTAINERNAME=akscs
-$TFSTATE_RG=tfstate
-```
-
-Create a Resource Group:
-```
-az group create --name $TFSTATE_RG --location $REGION
-```
-
-Create a Storage Account:
-```
-az storage account create -n $STORAGEACCOUNTNAME -g $TFSTATE_RG -l $REGION --sku Standard_LRS
-```
-
-Create a Storage Container within the Storage Account:
-
-```
-az storage container-rm create --storage-account $STORAGEACCOUNTNAME --name $CONTAINERNAME
-```
+1. Login to the Azure subscription that you'll be deploying into with your credentials
+   ``` az login 
+       az account set --subscription <Your subscription ID> 
+   ```
+2. Create some variables to start with about where your storage account will live.
+    ```Powershell
```

Comment:
So this is the first time a user is going to be asked to use a shell.  On the prior step you said PowerShell OR bash.  Yet this won't work if someone selected Bash.  Would it make sense to simply convert all of this (repo) over to PowerShell exclusively.  The "mixed mode" is only going to lead to user frustration if they end up on the wrong path.  I'd rather lead with one solid choice, and let a user figure out the other path if they want.